### PR TITLE
Update Banner

### PR DESCRIPTION
**Pull request by DentVega**
- event logo and banner added in svg
- banner design updated
- Facebook and meetup urls changed
- Pixel 2, iphone 6/7/8 plus phone and web view tested

### CODE REVIEW

File: src/app/home/banner/banner.component.html [hunk 1]
```diff
@@ -4,28 +4,31 @@
     <div id="banner-container" class="container">
       <div class="row">
         <div class="col offset-m2 m8">
-          <img id="devfest-logo" src="assets/images/logo-devfest.svg" alt="DevFest">
-          <img id="cbba-logo" src="assets/images/logo-cochabamba2019.svg" alt="Cochabamba 2019">
+          <img id="iwt-logo" src="assets/images/logo-wtm.svg" alt="Iwt">
+          <h2 id="banner-title">
+            International Women’s Day 2020
```
Comment (combimauri) [inline src/app/home/banner/banner.component.html]:
```suggestion
            International Women's Day 2020
```

File: src/app/home/banner/banner.component.html [hunk 2]
```diff
@@ -4,28 +4,31 @@
     <div id="banner-container" class="container">
       <div class="row">
         <div class="col offset-m2 m8">
-          <img id="devfest-logo" src="assets/images/logo-devfest.svg" alt="DevFest">
-          <img id="cbba-logo" src="assets/images/logo-cochabamba2019.svg" alt="Cochabamba 2019">
+          <img id="iwt-logo" src="assets/images/logo-wtm.svg" alt="Iwt">
```
Comment (combimauri) [inline src/app/home/banner/banner.component.html]:
```suggestion
          <img id="iwt-logo" src="assets/images/logo-wtm.svg" alt="IWT">
```

File: src/app/home/banner/banner.component.html [hunk 3]
```diff
@@ -4,28 +4,31 @@
...
+            International Women’s Day 2020
+          </h2>
+          <h3 id="banner-date">Marzo 28</h3>
           <h6 id="banner-details">
-            <strong>GDG DevFest Cochabamba</strong> es un evento para desarrolladores liderado por la comunidad Google Developers
-            Group Cochabamba. Somos una comunidad centrada en compartir el conocimiento sobre tecnologías de Google.
-          </h6>
-          <h6 id="place-and-date">
-            Noviembre 29-30
+            <strong>El International Women’s Day (IWD)</strong> se celebra en diferentes regiones de todo el mundo,
```
Comment (combimauri) [inline src/app/home/banner/banner.component.html]:
```suggestion
            <strong>El International Women's Day (IWD)</strong> se celebra en diferentes regiones de todo el mundo,
```

File: src/app/home/banner/banner.component.scss [hunk 4]
```diff
@@ -8,29 +8,38 @@
...
 
-#devfest-logo {
-  max-width: 100%;
-  margin-bottom: 20px;
+#banner-date {
+  width: 100%;
+  height: 34px;
+  font-size: 28px;
+  text-align: center;
+  color: #ff7976;
```
Comment (combimauri) [inline src/app/home/banner/banner.component.scss]: Place this color in styles.colors

File: src/app/layout/navbar/navbar.component.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <header>
-  <nav class="transparent">
+  <nav class="white">
```
Comment (combimauri) [inline src/app/layout/navbar/navbar.component.html:2]: Transparent style is used when page is at the top, so it starts changing when you scroll down. If you want to change the color of the banner, you have to change it in this element: `<div id="navbar-background"></div>`

File: src/styles.scss [hunk 6]
```diff
@@ -27,6 +27,10 @@ strong {
   font-weight: 700;
 }
 
+.secondary-btn {
+  background-color: $btn-color !important;
```
Comment (combimauri) [inline src/styles.scss]: Have you considered changing the value of `$secondary-color` instead of creating a new class?

**Review by combimauri — CHANGES_REQUESTED** [review]
Please add screenshots of your changes in the PR

**Comment by DentVega** [comment]
Captura de Pantalla 2020-03-13 a la(s) 10 17 27

Captura de Pantalla 2020-03-13 a la(s) 10 59 08

corrections finished

**Review by combimauri — APPROVED** [review]

**Pull request merged**